projects/sandbox/src/app/app.component.html

```diff
@@ -3,7 +3,7 @@
     <div class="header-container">
       <div class="metis-logo">
         <span class="phase-flag beta" tabindex="-1">BETA</span>
-        <a href="/" class="logo"></a>
+        <a href="/" aria-label="Europeana Metis Sandbox" class="logo"></a>
```

Comment:
Does this fix the issue with clicking on Europeana logo not working?

Reply:
Making the url a slash rather than a Router command would have fixed that specific issue, but the problem was deeper, and that bug was due to Angular.Location losing track of the current url due to some logic I fixed as part of MET-4266.

The Aria-Label is needed to stop Axe complaining about the link not containing any text.